Makefile

```diff
@@ -102,3 +102,13 @@ else
 endif
 
 verify: verify-gofmt
+
+sync-manifests:
```

Comment:
This may not work for 4.3 if there is API change in 4.4 development cycle. where only 4.4 needs to be updated.

Reply:
Fiixed